docs/Systems/Tailwind/Overview.stories.mdx

```diff
@@ -12,8 +12,17 @@ import AnatomyDiagram from './assets/tailwind-anatomy.jpg';
 This is an early release of the Kaizen Tailwind Preset - the API may change and evolve as we receive feedback from our teams. We will endeavour to assist where possible if classes or utilities are updated in the future.
 
 ## What our package does
-The `@kaizen/tailwind` package is essentially a custom Tailwind preset. For more information about Tailwind presets, see their docs [here](https://tailwindcss.com/docs/presets). This package does not change how Tailwind works; it only changes the suffixes that can be attached to utilities.  
-In the following diagram, the modifier `md:` and the utility `font-` are part of Tailwind. The suffix `family-heading` is defined by our preset, and represents one of our font-family design tokens.  
+The `@kaizen/tailwind` package is essentially a custom Tailwind preset. For more information about Tailwind presets, see their docs [here](https://tailwindcss.com/docs/presets). This package does not change how Tailwind works; it only defines the suffixes that can be applied when constructing utility classes.
+The following diagram shows the construction of a utility class that adds a background of `blue-400` on hover. The modifier `hover:` and the prefix `bg-` are part of Tailwind by default. The suffix `gray-400` is defined by our preset, and represents one of our color design tokens.  
 
 
-<img src={AnatomyDiagram} />
\ No newline at end of file
+<img src={AnatomyDiagram} />
+
+If the file containing the above utility class is processed by Tailwind, the following CSS rule will be generated, and added to the document:
+
+```css
+.hover\:bg-blue-400:hover {
+  --tw-bg-opacity: 1;
+  background-color: rgb(0 139 214 / var(--tw-bg-opacity))
+}
+```
```

Comment:
I think unnecessary detail, (I hope) the audience isn't here to learn how TW works 😄

Reply:
Cool, removed now

Reply:
Which is probably on purpose, cause I'd bet that the average TW user doesn't care and/or TW themselves want people to not care about this detail.